docs/container.md

```diff
@@ -171,6 +171,23 @@ printf('Container\'s name is $s', $container->getName());
 printf('Container\'s exit code is %d', $container->getExitCode());
 ```
 
+## Copy files or folders from a container
+
+File and folders can be downloaded as a tar file from the container. The copy() function allows a folder/file to be downloaded as tar stream and copyToDisk() saves the result as a tar file.
+In the following example the /etc/default folder from a ubuntu container called vanilla2 is downloaded to /tmp/resource.tar.
```

Comment:
1. [ ] *a ubuntu container* -> **an** ubuntu container

God I wish I could do the edit myself instead of asking you to do them, GitHub really needs to allow multiple people to contribute commits to a PR...